sonar-project.properties

```diff
@@ -5,8 +5,8 @@ sonar.exclusions=**/*_test.go,**/vendor/**,**/java/**,go/test/endtoend/**,**/*.p
 sonar.tests=.
 sonar.test.inclusions=**/*_test.go
 sonar.test.exclusions=**/vendor/**
-
-
+sonar.go.coverage.reportPaths=/tmp/*.out
+#sonar.coverage.dtdVerification=false 
```

Comment:
If we want to leave this in, but commented out, there should be a comment explaining why.